mailpoet/lib/Config/Installer.php

```diff
@@ -27,6 +28,14 @@ public function init() {
     WPFunctions::get()->addFilter('plugins_api', [$this, 'getPluginInformation'], 10, 3);
   }
 
+  public function generatePluginActivationUrl(string $plugin): string {
+    return WPFunctions::get()->adminUrl('plugins.php?' . implode('&', [
+      'action=activate',
+      'plugin=' . urlencode($plugin),
+      '_wpnonce=' . wp_create_nonce('activate-plugin_' . $plugin),
```

Comment:
When calling the WordPress functions we always do it using the `WPFunctions` proxy class. Could you please create a method in there called `wpCreateNonce` and use it instead of directly calling the function here?

Reply:
@pavel-mailpoet Noted 👍